extension/sidebar-pane.js

```diff
@@ -38,13 +43,28 @@ async function _updateSubtree(selectedNode) {
   progressEl.textContent = "Getting web compatibility issues";
   issueListEl.innerHTML = "";
   _recursiveNodesIssuesRendering(0, nodesInSubtree, false, issueListEl, []).then(() => {
+    if (!_isCurrentProcess(issueListEl)) {
+      return;
+    }
+
     subtreeEl.classList.remove("processing");
     if (!issueListEl.querySelector("li")) {
       _renderNoIssue(issueListEl);
     }
   });
 }
 
+function _getNewULElement(selector) {
+  const oldElement = document.querySelector(selector);
+  const newElement = document.createElement("ul");
+  oldElement.parentElement.replaceChild(newElement, oldElement);
+  return newElement;
+}
+
+function _isCurrentProcess(listEl) {
```

Comment:
I think this function's name could be better. The term `process` means something that I don't think maps to this.
Maybe `_isCurrentlyRendering` would be better?